docs/react-v9/contributing/component-implementation-guide.md

```diff
@@ -0,0 +1,264 @@
+
+Run the following command (replace `MYCOMPONENT` with the appropriate package name):
+
+```bash
+yarn nx generate prepare-initial-release --project @fluentui/react-MYCOMPONENT-preview --phase=preview
+```
+
+# Phase 3: Validation and Stable Release
+
+Once the component has been released as unstable, you should perform additional validation and complete any tests and documentation that was not finished before.
+
+## Tests
+
+- **Conformance tests**
+  - Call `isConformant` from your components' tests.tsx file.
+- **Unit tests**
+  - Render your component with [`@testing-library/react`](https://testing-library.com/docs/react-testing-library/intro) for the tests.
+  - Use [queries](https://testing-library.com/docs/queries/about) like `getByRole` to find elements in the rendered DOM tree, and check that their attributes are as expected.
+  - Avoid using snapshot tests.
+  - When deciding what tests to write, generally the tests should cover all functionality in your component's state hook (`use{Component}`)
+  - Follow examples from other components for more guidance, such as [Button.test.tsx](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-button/src/components/Button/Button.test.tsx)
+- **Visual Regression Tests**
+  - VR tests ensure that the components visuals do not change unexpectedly.
+  - Add stories to the `vr-tests-react-components` app, for example [Button.stories.tsx](https://github.com/microsoft/fluentui/blob/master/apps/vr-tests-react-components/src/stories/Button.stories.tsx)
+  - When deciding what tests to write, generally the tests should cover all states in your component's styling hook (`use{Component}Styles`).
+- **Bundle Size Tests**
+  - Example: [Button.fixture.js](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-button/bundle-size/Button.fixture.js)
+- **Performance Tests**
+  - Example: [Button.tsx](https://github.com/microsoft/fluentui/blob/master/apps/perf-test-react-components/src/scenarios/Button.tsx)
+- **Manual Accessibility Tests**
+  - For more info see [Manual Accessibility Review Checklist](https://github.com/microsoft/fluentui/wiki/Manual-Accessibility-Review-Checklist)
+
+## Validation
+
+The converged component can't be considered fully complete until it is validated in a real product. Work with a partner team to make sure that they can use it in their product.
+
+## Bug Bash
+
+Organize a bug bash with other Fluent UI crews. This is the last chance to verify architecture consistency across the whole library and raise awareness among all crews. The feedback should be addressed before the stable release. There should be a bug bash for each major Time Zone (e.g. one for US and one for Europe). _Tip: Look for a volunteer from another Time Zone to schedule and run the bug bash for your component._
```

Comment:
Would be great if we could have an email/teams groups or leads to contact/assign for bug bashes, AFAIK we must hit a bug bash with both Redmond and Prague offices to be considered complete, but who/when/how is not clear unless you are aware of that ahead of time.